TestApp/app/src/main/java/com/example/testapp/viewmodel/MovieViewModel.kt

```diff
@@ -0,0 +1,2 @@
+package com.example.testapp.viewmodel
+
```

Comment:
this movie view model should be a part of the movie package

Reply:
done same as login